pyiron_base/project/delayed.py

```diff
@@ -57,7 +57,7 @@ def get_graph(
     Returns:
         dict, list: dictionary of nodes and list of edges
     """
-    if isinstance(obj, DelayedObject):
+    if isinstance(obj, DelayedObject) and obj_name is not None:
```

Comment:
```suggestion
    if isinstance(obj, DelayedObject) and obj_name is None:
```

Sorry I made a mistake XD